pkg/policy/resolve_test.go

```diff
@@ -535,20 +533,22 @@ func (ds *PolicyTestSuite) TestMapStateWithIngressWildcard(c *C) {
 	}
 	wg := &sync.WaitGroup{}
 	testSelectorCache.UpdateIdentities(added1, nil, wg)
+	// Cleanup the identities from the testSelectorCache
+	defer testSelectorCache.UpdateIdentities(nil, added1, wg)
```

Comment:
Why was this defer statement moved from TestMapStateWithIngress to TestMapStateWithIngressWildcard?

Reply:
The defer statement here is to make sure that the cleanup is done, so that there is no side effect for other tests. IMO, this is the missing step in existing tests, and just show up as go test and gocheck is having different order of test execution.